server/src/http/background_send_metrics.rs

```diff
@@ -87,7 +87,7 @@ pub async fn send_metrics_one_shot(
                 } else {
                     feature_refresher
                         .unleash_client
-                        .send_batch_metrics(batch.clone())
+                        .send_batch_metrics(batch.clone(), None)
```

Comment:
first sync is not from the scheduler so no interval passed

Reply:
I was hesitant to do this since it's not called from the scheduler (negligible one-off call as opposed to regular scheduled calls) + I'd need to drag the interval through server function calls (main -> clean_shutdown -> send_metrics_one_shot -> send_batch_metrics). I'm willing to re-visit this decision though :)